components/Issuer/IndyWalletConnectionSetUp.vue

```diff
@@ -15,7 +15,7 @@ const generateQRCodeForConnection = () => {
     console.log(`invitation.....${invitationURL}`)
     invitationLink.value = invitationURL
     WalletConnectionID = connectionID
-    emit('addToLog', '[Issuer] Creating connection QRCode')
+    emit('addToLog', 'Creating connection QRCode', 'Issuer', null)
```

Comment:
Can you omit the last parameter and handle it with a `?` in the method definition? Then this and some of the other calls would become

```suggestion
    emit('addToLog', 'Creating connection QRCode', 'Issuer')
```